Fix kdepim/certmanager compilation with --enable-final,
	Ingmar Vanhassel <ingmar@gentoo.org>

diff -Naur kdepim-3.5.9.orig/certmanager/lib/cryptplug.h kdepim-3.5.9/certmanager/lib/cryptplug.h
--- kdepim-3.5.9.orig/certmanager/lib/cryptplug.h	2005-09-10 10:23:51.000000000 +0200
+++ kdepim-3.5.9/certmanager/lib/cryptplug.h	2008-02-21 15:38:24.000000000 +0100
@@ -408,7 +408,7 @@
    \note This function <b>must</b> be implemented by each plug-in using
    this API specification.
 */
-bool hasFeature( Feature );
+bool hasFeature( ::Feature );
 
 /*! \ingroup groupSignCryptAct
    \brief Information record returned by signing and by encrypting
diff -Naur kdepim-3.5.9.orig/certmanager/lib/cryptplugwrapper.h kdepim-3.5.9/certmanager/lib/cryptplugwrapper.h
--- kdepim-3.5.9.orig/certmanager/lib/cryptplugwrapper.h	2008-02-13 10:39:29.000000000 +0100
+++ kdepim-3.5.9/certmanager/lib/cryptplugwrapper.h	2008-02-21 16:17:41.000000000 +0100
@@ -718,7 +718,7 @@
 
         \return  whether the relative feature is implemented or not
     */
-    bool hasFeature( Feature );
+    bool hasFeature( ::Feature );
 
 
     /* \ingroup groupSignAct
